tests/text/test_rouge.py

```diff
@@ -0,0 +1,205 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+import pytest
+import torch
+from torch import tensor
+
+from torchmetrics.functional.text.rouge import rouge_score
+from torchmetrics.text.rouge import ROUGEScore
+from torchmetrics.utilities.imports import _NLTK_AVAILABLE, _ROUGE_SCORE_AVAILABLE
+
+if _ROUGE_SCORE_AVAILABLE:
+    from rouge_score.rouge_scorer import RougeScorer
+    from rouge_score.scoring import BootstrapAggregator
+else:
+    RougeScorer, BootstrapAggregator = object, object
+
+ROUGE_KEYS = ("rouge1", "rouge2", "rougeL", "rougeLsum")
+
+PRECISION = 0
+RECALL = 1
+F_MEASURE = 2
+
+SINGLE_SENTENCE_EXAMPLE_PREDS = 'The quick brown fox jumps over the lazy dog'
```

Comment:
@SkafteNicki. Should we have a test with different sentences on each rank compared to 1 rank with all sentences ?

Reply:
That would be the optimal case I think

Reply:
@tchaton after discussion with @karthikrangasai we agree on doing a follow up PR where we make a specialized `TextTester` class (that should inherent from the `MetricTester` class that all other metrics use) such that we can get more robust tests for all the new text metrics

Reply:
I have opened an issue for the same idea in #411